test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -26,6 +27,17 @@
             [clojure.set :refer [difference]]
             [puppetlabs.puppetdb.test-protocols :as test-protos]))
 
+(defn ordered-matches?
```

Comment:
Do we need to have an extensions PR that removes that from extensions and uses this one?